controller/src/main/java/org/jboss/as/controller/AbstractAttributeDefinitionBuilder.java

```diff
@@ -334,6 +334,51 @@ public BUILDER addAlternatives(String... alternatives) {
     }
 
     /**
+     * Removes {@link AttributeDefinition#getAlternatives() names of alternative attributes} from the set of those that should not
+     * be defined if this attribute is defined.
+     * @param alternatives the attribute names
+     * @return a builder that can be used to continue building the attribute definition
+     */
+    public BUILDER removeAlternatives(String... alternatives) {
```

Comment:
Please make this final. A lot of methods in this class should have been final but aren't, but since this is a new one let's lock it down.